Automattic/wp-calypso
# Sensei Tailored Flow: Update setup step design

#### Proposed Changes

* Update "Set up your course site" step, address design review comments
  * Adjust spacing and layout
  * Update preview images to latest version

#### Testing Instructions

* Start a new Sensei site, log in if needed: http://calypso.localhost:3000/setup/sensei/
* View the 'Set up your course site' (the first) step in various device sizes and check that all looks good and matches design

#### Pre-merge Checklist

N/A

#### Screenshots

![image](https://user-images.githubusercontent.com/176949/214192601-dfb51d6e-1857-4a59-9244-e90f24dc4732.png)
![image](https://user-images.githubusercontent.com/176949/214192616-71a6f36d-d64d-4cee-a5ae-098fed1bae12.png)
<img src=https://user-images.githubusercontent.com/176949/214192665-98540054-00d6-48ca-afab-5d1c42251397.png width='400' /><img src=https://user-images.githubusercontent.com/176949/214192712-efc5ed41-cdca-4963-8657-5572ef1ca119.png width='400'  />
